src/fireedge/src/modules/constants/translates.js

```diff
@@ -0,0 +1,2506 @@
+  Wed: 'Wed',
+  Wednesday: 'Wednesday',
+  Thu: 'Thu',
+  Thursday: 'Thursday',
+  Fri: 'Fri',
+  Friday: 'Friday',
+  Sat: 'Sat',
+  Saturday: 'Saturday',
+  Sun: 'Sun',
+  Sunday: 'Sunday',
+  Daily: 'Daily',
+  Weekly: 'Weekly',
+  Monthly: 'Monthly',
+  Yearly: 'Yearly',
+  Hourly: 'Hourly',
+  Years: 'years',
+  Months: 'months',
+  Weeks: 'weeks',
+  Days: 'days',
+  Hours: 'hours',
+  Minutes: 'minutes',
+  Seconds: 'seconds',
+  EachHours: 'Each %s hours',
+  AfterTimes: 'After %s times',
+  Today: 'Today',
+  Periodic: 'Periodic',
+  Time: 'Time',
+  TimeAfterTheVmIsInstantiated: 'Time after the VM is instantiated',
+  PeriodType: 'Period type',
+  GranularityOfAction: 'Granularity of the action',
+  DayOfWeek: 'Day of week',
+  DayOfMonth: 'Day of month',
+  DayOfYear: 'Day of year',
+  EachXHours: "Every 'x' hours",
+  EndType: 'End type',
+  DaysBetween0_6: 'Days should be between 0 (Sunday) and 6 (Saturday)',
+  DaysBetween1_31: 'Days should be between 1 and 31',
+  DaysBetween0_365: 'Days should be between 0 and 365',
+  HoursBetween0_168: 'Hours should be between 0 and 168',
+  WhenDoYouWantThisActionToStop: 'When do you want this action to stop?',
+  HowManyTimesDoYouWantThisActionToExecute:
+    'How many times do you want this action to execute?',
+  NumberOfRepetitions: 'Number of Repetitions',
+  In: 'in',
+  WarningBefore: 'Warning before',
+  Never: 'Never',
+  Repetition: 'Repetition',
+  Date: 'Date',
+
+  /* footer */
+  MadeWith: 'Made with',
+  NotOfficiallySupport: 'Not Officially Supported',
+  OfficiallySupport: 'Officially Supported',
+
+  /* dashboard */
+  InTotal: 'In Total',
+  Used: 'Used',
+  MyDashboard: 'My Dashboard',
+  Greetings: 'Hey, ',
+  CreateVM: 'Create VM',
+  LimitProfileImage: 'the image must be less than 2 MB',
+
+  /* login */
+  LogIn: 'Login in your account:',
+  Username: 'Username',
+  Password: 'Password',
+  ConfirmPassword: 'Confirm Password',
+  PasswordsMustMatch: 'Passwords must match',
+  ChangePassword: 'Change password',
+  ChangePasswordSuccess: 'Succesfully changed password',
+  ChangePasswordAdminWarning:
+    'Users oneadmin and serveradmin can not change theirs passwords using Sunstone.',
+  ChangePasswordAdminWarningLink:
+    'See OpenNebula documentation to get more details.',
+  ChangePasswordLdapWarning:
+    'Users with authentication driver LDAP can not change theirs passwords using Sunstone.',
+  Token2FA: '2FA Token',
+  GetAuthenticatorApp: 'Get authenticator app: ',
+  ScanThisQr: 'Scan this Qr',
+  EnterVerificationCode: 'Enter Verification Code',
+  TwoFactorAuthentication: 'Two Factor Authentication',
+  RegisterAuthenticationApp: 'Register authentication App',
+  AuthenticatorAppSunstone: 'Authenticator app (HOTP) [Legacy]',
+  AuthenticatorApp: 'Authenticator app (HOTP)',
+  KeepLoggedIn: 'Keep me logged in',
+  KeepLast: 'Keep Last',
+  KeepAliveID: 'Keepalive service ID',
+  KeepAlivePass: 'Keepalive password',
+  Credentials: 'Credentials',
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential hardcoded credential detected. This code may contain sensitive data such as passwords or API keys embedded directly in the source.  Hardcoded credentials can be extracted and misused, leading to unauthorized access to systems or data breaches.  To remediate this, store secrets in environment variables or use a secrets management tool like AWS Secrets Manager, Azure Key Vault, or HashiCorp Vault.  Avoid committing credentials to version control. For best practices, refer to - https://owasp.org/www-community/vulnerabilities/Use_of_hard-coded_password